packages/components/src/search-list-control/style.scss

```diff
@@ -104,6 +103,11 @@
 			background: $gray-100;
 		}
 
+		&:active,
+		&:focus {
+			box-shadow: none;
```

Comment:
This hides the `:focus`/`:active` box shadow, which was looking a bit off. We already have `:hover` styles, so I think that's enough for a11y:

| Before | After |
| --- | --- |
| ![imatge](https://user-images.githubusercontent.com/3616980/102516216-e5363200-408e-11eb-91b8-dec878bfd19f.png) | ![imatge](https://user-images.githubusercontent.com/3616980/102516248-ed8e6d00-408e-11eb-9f11-c2087a257eca.png) |